src/tool/hpcrun/safe-sampling.h

```diff
@@ -167,6 +167,10 @@ hpcrun_safe_enter_async(void *pc)
 static inline void
 hpcrun_safe_exit(void)
 {
+  if (! hpcrun_is_initialized() || ! hpcrun_td_avail()) {
+    return 0;
```

Comment:
Since the return type of this function is void. I'll suggest you just use `return;`

Reply:
Whoops, my bad. I guess that's called copy-pasting. 🙈